clients/cloud/rust/src/utils.rs

```diff
@@ -0,0 +1,53 @@
+extern crate clap;
+extern crate rdkafka;
+
+use clap::{App, Arg};
+use rdkafka::config::{ClientConfig, RDKafkaLogLevel};
+use std::boxed::Box;
+use std::fs::File;
+use std::io::BufRead;
+use std::io::BufReader;
+
+pub fn get_config() -> Result<(String, ClientConfig), Box<std::error::Error>> {
+    let matches = App::new("rust client example")
+        .version(option_env!("CARGO_PKG_VERSION").unwrap_or(""))
+        .arg(
+            Arg::with_name("config")
+                .help("path to confluent cloud config file")
+                .long("config")
+                .takes_value(true)
+                .required(true),
+        )
+        .arg(
+            Arg::with_name("topic")
+                .help("test topic to use")
+                .long("topic")
+                .takes_value(true)
+                .required(true),
+        )
+        .get_matches();
+
+    let mut kafka_config = ClientConfig::new();
+    kafka_config.set_log_level(RDKafkaLogLevel::Debug);
+
+    let file = File::open(matches.value_of("config").ok_or("error parsing config")?)?;
+    for line in BufReader::new(&file).lines() {
+        let cur_line = line?;
+        if cur_line.starts_with('#') || cur_line.chars().count() < 1 {
+            continue;
+        }
+        let key_value: Vec<&str> = cur_line.split("=").collect();
+        kafka_config.set(
```

Comment:
will this propagate errors?

Reply:
anywhere where there is an error, a result is returned and errors are propogated